### DISCUSSION TITLE
remove database while uninstalling jans

### DISCUSSION THREAD

**Discussion by manojs1978** [JanssenProject/jans#7575] [Q&A] [answered]
is it possible to remove database while uninstalling jans?

**Comment by ossdhaval** [comment, marked as answer]
Hi @manojs1978 

Removing the datastore is a manual process as it is not recommended to uninstall it with the Janssen server software. Here is the [issue](https://github.com/JanssenProject/jans/issues/5939) where you can see past discussion on this topic.
